web/src/main/java/org/springframework/security/web/DefaultSecurityFilterChain.java

```diff
@@ -48,10 +50,13 @@ public DefaultSecurityFilterChain(RequestMatcher requestMatcher, Filter... filte
 
 	public DefaultSecurityFilterChain(RequestMatcher requestMatcher, List<Filter> filters) {
 		if (filters.isEmpty()) {
-			logger.info(LogMessage.format("Will not secure %s", requestMatcher));
+			logger.debug(LogMessage.format("Will not secure %s", requestMatcher));
 		}
 		else {
-			logger.info(LogMessage.format("Will secure %s with %s", requestMatcher, filters));
+			logger.debug(LogMessage.format("Will secure %s with filters: %s", requestMatcher,
+					filters.stream()
```

Comment:
To remain in line with https://github.com/spring-projects/spring-security/issues/7154 will you please use a for loop instead?

Reply:
Thank you for the review. I will update it to use a for loop as suggested.